### ISSUE TITLE
Compromised Password shows already deleted entries

### ISSUE THREAD

**Issue by nylocx** [bitwarden/server#988]
If I request a report about compromised passwords a lot of passwords are shown twice. One of them is no longer present in my vault but still shown in the report. These are all passwords from a failed Import that I had to delete because of some encoding f**k up. Why are these passwords still there and shown as not being in a Folder, but if I list passwords not being in a folder I get an empty list.
I am still able to view the passwords of this deleted accounts, so they are definitely still there - somewhere -.

For reference I use a  self hosted bitwarden with a license and checked for updates this morning.

Kind regards,
Alex

**Comment by clayadams5226** [comment]
You are correct. At this time the report includes items in your trash. Once deleted from there you should get your desired result.

**Comment by nylocx** [comment]
Would be nice it those are marked in a way that they are already deleted.
